sci-chemistry/autodock/autodock-4.2.6-r1.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools readme.gentoo-r1
+
+MY_PN="autodocksuite"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A suite of automated docking tools"
+HOMEPAGE="http://autodock.scripps.edu/"
+SRC_URI="http://autodock.scripps.edu/downloads/autodock-registration/tars/dist$(ver_rs 1- '')/${MY_P}-src.tar.gz"
```

Comment:
Could you also update these url's to use https:
```
DeadUrl: version 4.2.6-r1: SRC_URI: 404 Client Error: Not Found for url: https://autodock.scripps.edu/downloads/autodock-registration/tars/dist426/autodocksuite-4.2.6-src.tar.gz: http://autodock.scripps.edu/downloads/autodock-registration/tars/dist426/autodocksuite-4.2.6-src.tar.gz
HttpsUrlAvailable: version 4.2.6-r1: HOMEPAGE: HTTPS url available: http://autodock.scripps.edu/ -> https://autodock.scripps.edu/
```

Reply:
Done. Please suggest how we are going to compress the patch. Maybe we can refer what soap did here, https://github.com/gentoo/gentoo/pull/32041#issuecomment-1656761534.

Reply:
Uploaded the patch to my devspace and amended the commit to fetch it from there.

Merged, Thanks :+1: